src/main/java/mate/controller/driver/AddDriverController.java

```diff
@@ -23,10 +23,10 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
     @Override
     protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             throws IOException, ServletException {
-        String name = req.getParameter("name");
-        String licenceNumber = req.getParameter("licence_number");
-        Driver driver = new Driver(name, licenceNumber);
+        Driver driver = new Driver(req.getParameter("name"),
+                req.getParameter("licence_number"), req.getParameter("login"),
+                req.getParameter("password"));
         driverService.create(driver);
-        resp.sendRedirect("/drivers/add");
+        req.getRequestDispatcher("/login").forward(req, resp);
```

Comment:
better to use resp.sendRedirect because after forwarding the parameters you send are saved in the browser and if refreshing the page you will send them again to server